<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editorSeleccionArmasHerramientas</class>
 <widget class="QDialog" name="editorSeleccionArmasHerramientas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1171</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2" alignment="Qt::AlignLeft">
    <widget class="QLabel" name="label_10">
     <property name="minimumSize">
      <size>
       <width>97</width>
       <height>48</height>
      </size>
     </property>
     <property name="text">
      <string>Infinite Ammo</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QCheckBox" name="BazookaIA">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Infinite Ammo</string>
     </property>
    </widget>
   </item>
   <item row="1" column="9">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>Infinite Ammo</string>
     </property>
    </widget>
   </item>
   <item row="3" column="7">
    <widget class="QLabel" name="dinamita">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>W4_Dynamite.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="7">
    <widget class="QSpinBox" name="municionGranadaV">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </item>
   <item row="8" column="4">
    <widget class="QCheckBox" name="Teletransportador">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Banana</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QCheckBox" name="Mortero">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="8" column="10" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="5" column="11">
    <widget class="QCheckBox" name="Bate">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="8">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Green grenade</string>
     </property>
    </widget>
   </item>
   <item row="3" column="10">
    <widget class="QLabel" name="bate">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>Baseballbat.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="8">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Dynamite</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="Banana">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QSpinBox" name="municionBanana">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </item>
   <item row="0" column="10">
    <widget class="QLabel" name="granadaR">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>Redgrenade.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="11">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Baseball bat</string>
     </property>
    </widget>
   </item>
   <item row="2" column="11">
    <widget class="QCheckBox" name="GranadaR">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QSpinBox" name="municionMortero">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QSpinBox" name="municionBazooka">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </item>
   <item row="5" column="7">
    <widget class="QSpinBox" name="municionDinamita">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QCheckBox" name="Aereo">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QLabel" name="tele">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>IconTeleport.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="10">
    <widget class="QSpinBox" name="municionBate">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </item>
   <item row="2" column="10">
    <widget class="QSpinBox" name="municionGranadaR">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QSpinBox" name="municionGranadaS">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Air attack</string>
     </property>
    </widget>
   </item>
   <item row="5" column="8">
    <widget class="QCheckBox" name="Dinamita">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="mortero">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>Mortar1.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="Bazookka">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QLabel" name="granadaV">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>W4_Grenade.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QCheckBox" name="GranadaS">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="8">
    <widget class="QCheckBox" name="GranadaV">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="7" column="4">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Teletransportation</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="Bazooka">
     <property name="text">
      <string>Bazooka</string>
     </property>
    </widget>
   </item>
   <item row="1" column="11">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Red grenade</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Mortar</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="aereo">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>W4_Airstrike.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Holy grenade</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="banana">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>Bananabomb.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QSpinBox" name="municionAereo">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="bazooka_2">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>Bazooka.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="3">
    <widget class="QSpinBox" name="municionTeletransportador">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="granadaS">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>Holy_Grenade.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="12">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>Infinite Ammo</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>Infinite Ammo</string>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QLabel" name="label_15">
     <property name="text">
      <string>Infinite Ammo</string>
     </property>
    </widget>
   </item>
   <item row="4" column="9">
    <widget class="QLabel" name="label_16">
     <property name="text">
      <string>Infinite Ammo</string>
     </property>
    </widget>
   </item>
   <item row="4" column="12">
    <widget class="QLabel" name="label_17">
     <property name="text">
      <string>Infinite Ammo</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QLabel" name="label_18">
     <property name="text">
      <string>Infinite Ammo</string>
     </property>
    </widget>
   </item>
   <item row="7" column="5">
    <widget class="QLabel" name="label_19">
     <property name="text">
      <string>Infinite Ammo</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QCheckBox" name="MortaIA">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="9">
    <widget class="QCheckBox" name="GreenGrenadeIA">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="12">
    <widget class="QCheckBox" name="RedGrenadeIA">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QCheckBox" name="BananaIA">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="5">
    <widget class="QCheckBox" name="HGrenadeIA">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="9">
    <widget class="QCheckBox" name="DynamiteIA">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="12">
    <widget class="QCheckBox" name="BatIA">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QCheckBox" name="AIrIA">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="8" column="5">
    <widget class="QCheckBox" name="TeleIA">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>editorSeleccionArmasHerramientas</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>editorSeleccionArmasHerramientas</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
